src/atoms/atomgraph.jl

```diff
@@ -132,7 +132,7 @@ end
 
 # pretty printing, long version
 function Base.show(io::IO, ::MIME"text/plain", ag::AtomGraph)
-    st = "AtomGraph $(ag.id) with $(nv(ag.graph)) nodes, $(ne(ag.graph)) edges\n   atoms: $(ag.elements)\n"
+    st = "AtomGraph $(ag.id) with $(nv(ag.graph)) nodes, $(ne(ag.graph)) edges\n   atoms: $(ag.elements)"
     print(io, st)
```

Comment:
If multiple `AtomGraph`s were being printed, wouldn't this result in them not being printed line by line? (as the `\n` is removed)

Reply:
As it was it was adding an extra newline (I'm guessing `\n` gets appended by default)...with this, the result is like this:
```
julia> ags
8-element Vector{Union{Missing, AtomGraph}}:
 AtomGraph test/test_data/strucs/methane with 5 nodes, 10 edges
 AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges
 AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges
 AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges
 AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges
 AtomGraph test/test_data/strucs/mp-224 with 6 nodes, 9 edges
 missing
 AtomGraph test/test_data/strucs/testgraph with 3 nodes, 3 edges

julia> print(ags)
Union{Missing, AtomGraph}[AtomGraph test/test_data/strucs/methane with 5 nodes, 10 edges, AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges, AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges, AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges, AtomGraph test/test_data/strucs/mp-195 with 4 nodes, 6 edges, AtomGraph test/test_data/strucs/mp-224 with 6 nodes, 9 edges, missing, AtomGraph test/test_data/strucs/testgraph with 3 nodes, 3 edges]
```

which is stylistically consistent with the way other lists are handled:
```
julia> l = [k for k in 1:5]
5-element Vector{Int64}:
 1
 2
 3
 4
 5

julia> print(l)
[1, 2, 3, 4, 5]
```